cms/static/js/views/settings/advanced.js

```diff
@@ -109,7 +109,7 @@ var AdvancedView = ValidatingView.extend({
             success : function() {
                 self.render();
                 var title = gettext("Your policy changes have been saved.");
-                var message = gettext("Please note that validation of your policy key and value pairs is not currently in place yet. If you are having difficulties, please review your policy pairs.");
+                var message = gettext("Take care with key and value formatting in Advanced Settings, as no validation is performed. If you are having difficulties, please review your policy pairs.");
```

Comment:
Take care  --> Be careful (less casual?)
Break into two sentences?
"Be careful with the formatting that you enter for key and value pairs. No validation is performed."